src/main/java/com/powsybl/openloadflow/network/impl/LfBranchImpl.java

```diff
@@ -85,6 +87,32 @@ private static LfBranchImpl createTransformer(TwoWindingsTransformer twt, LfBus
             piModel = new PiModelArray(models, ptc.getLowTapPosition(), ptc.getTapPosition());
         }
 
+        RatioTapChanger rtc = twt.getRatioTapChanger();
+        if (rtc != null && rtc.isRegulating()) {
```

Comment:
In this code rtc override ptc, so we don't support the case where we have both an active rtc and ptc. So we should either throw an exception or select one (which one?) and warn the user.

Reply:
I have tried to make some improvements and I have created a test but... I fall in powsybl-core exceptions `com.powsybl.iidm.network.ValidationException: 2 windings transformer 'T2wT': Only one regulating control enabled is allowed`